easybuild/easyconfigs/g/GAMESS-US/compddi.patch

```diff
@@ -0,0 +1,13 @@
+--- gamess.orig/ddi/compddi	2013-05-04 08:14:15.000000000 +1200
```

Comment:
I didn't know how to do that. Now I do, and I'll endeavour to add the comments in the coming week.